src/CMakeLists.txt

```diff
@@ -297,6 +306,28 @@ if(APPLE AND WITH_APP_BUNDLE)
                      COMMAND ${MACDEPLOYQT_EXE} ${PROGNAME}.app
                      WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/src
                      COMMENT "Deploying app bundle")
+
+  if(WITH_XC_BROWSER)
+    set(PROXY_BINARY_DIR "${CMAKE_BINARY_DIR}/src/proxy/keepassxc-proxy")
+    set(PROXY_APP_DIR "${PROGNAME}.app/Contents/MacOS/keepassxc-proxy")
+    add_custom_command(TARGET ${PROGNAME}
+                       POST_BUILD
+                       COMMAND ${CMAKE_COMMAND} -E copy ${PROXY_BINARY_DIR} ${PROXY_APP_DIR}
+                       WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/src
+                       COMMENT "Copying keepassxc-proxy inside the application")
+
+    add_custom_command(TARGET ${PROGNAME}
+                       POST_BUILD
+                       COMMAND ${CMAKE_INSTALL_NAME_TOOL} -change /usr/local/opt/qt/lib/QtCore.framework/Versions/5/QtCore "@executable_path/../Frameworks/QtCore.framework/Versions/5/QtCore" ${PROXY_APP_DIR}
```

Comment:
This might break non-Homebrew Qt installations (e.g. MacPorts).

Reply:
Yes. It's possible. AFAIK there's no way to get the exact framework location with cmake. So this is the current workaround. Maybe doing symbolic links to that location might work with MacPorts. I will do some testing.

Reply:
I think we should fix that in a separate PR.
The usual way to handle this is to use file() with several hints for possible locations.

Reply:
I agree.